node/internal.ts

```diff
@@ -487,7 +487,7 @@ function _tryGetExecutablePath(filePath: string, extensions: string[]): string {
                     // on Mac/Linux, test the execute bit
                     //     R   W  X  R  W X R W X
                     //   256 128 64 32 16 8 4 2 1
-                    if ((stats.mode & 1) == 1) {
+                    if ((stats.mode & 73) == 1) {
```

Comment:
It might be worth a comment out to the right like:

```
if ((stats.mode & 73) == 1) { // 73 = 64 + 8 + 1
```

or more explicit:
```
if ((stats.mode & (64 + 8 + 1)) == 1) {
```